wrapanapi/msazure.py

```diff
@@ -359,8 +359,50 @@ def list_stack(self, resource_group=None, days_old=0):
     def list_flavor(self):
         raise NotImplementedError('list_flavor not implemented.')
 
-    def list_network(self):
-        raise NotImplementedError('list_network not implemented.')
+    def list_networks(self):
+        self.logger.info("Attempting to List Azure Virtual Private Networks of the provider region")
```

Comment:
can we log self.region in all of these log messages so we have some clues as to why an entity didn't pop up if someone accidentally created it in the wrong region

```python
self.logger.info("Attempting to List Azure Virtual Private Networks in region '%s'", self.region)
```

Reply:
I will modify add this.